code/datums/jobs.dm

```diff
@@ -128,16 +128,18 @@
 		M.faction |= src.faction
 
 		SPAWN(0)
```

Comment:
Does anything break if this `SPAWN` isn't here? I vaguely remember implants being one of the things that did need that, but I can't remember for sure.

Reply:
SPAWN covers all these, will test o7
- [ ] regular implants
- [ ] sec clone disk
- [ ] mobcritter implants
- [ ] latejoin radio host (test `special_spawn_location` / custom name)
- [ ] bioeffect/mutrace

Reply:
This works without the spawn with local testing. Unsure if this might be a problem with the large number of round-start uses at one time?